pkg/bootstrap/bootstrap_config_test.go

```diff
@@ -66,7 +66,7 @@ var (
 func TestGolden(t *testing.T) {
 	cases := []struct {
 		base                       string
-		labels                     map[string]string
+		envVars                    map[string]string
```

Comment:
@douglas-reid in another PR I changed this code , can you confirm automatic merge is doing the right thing?  Or rebase...

Reply:
if your PR went in recently, this probably needs a rebase. It has been open for awhile now.

Reply:
I rebased and force-pushed, but this code did not change.

Reply:
ok, now I see the new code in your PR :-)
so perhaps the auto merge was working.